var/spack/repos/builtin/packages/c-blosc2/package.py

```diff
@@ -11,11 +11,13 @@ class CBlosc2(CMakePackage):
        other bells and whistles"""
 
     homepage = "http://www.blosc.org"
+    url      = "https://github.com/Blosc/c-blosc2/archive/refs/tags/v2.0.0.rc1.tar.gz"
     git      = "https://github.com/Blosc/c-blosc2.git"
 
     maintainers = ['ax3l']
 
     version('develop', branch='master')
+    version('2.0.0', sha256='c30b72af5446f052bad1791000e5a44d156c96b0e39b4bc8e9f97a013c7d1b69')
```

Comment:
This is not `v2.0.0` but `v2.0.0-rc1` I assume that it's planned to release the major version later.

Reply:
Correct. Given that it needed 5 years, 8th of Jan  2016, from alpha 2 to RC1, I would expect it in at least a year or two. Therefore, I have included it here as there was just git.

Reply:
Fair enough, even though `beta5` and `rc1` are just 2 weeks apart. I'm fine with whatever @ax3l thinks is fine with respect to the version tag.

Reply:
Nah, you saw it wrongly...  It is 1 year and 2 weeks. :)

Reply:
:laughing: Correct, I missed that detail.

Reply:
Yep, I know the RC is open for a bit but that does not mean there won't be a full release :)
Let's name it `2.0.0.rc1`? :-)

Reply:
Fixed :)